src/App.jsx

```diff
@@ -41,15 +40,13 @@ function App() {
   }, []);
 
   return (
-    <AuthProvider>
```

Comment:
Yeah that was the initial reason for the switch, 

But then after looking into it more. You're actually supposed to have your AuthProvider at the root of the application.

So main is where it is supposed to be instead of app.jsx